django_project/core/base_static/css/stylesheet.css

```diff
@@ -835,3 +842,7 @@ nav#sidebar ul li a:hover {
     background-color: var(--c-smoke);
 }
 
+.input-invalid {
+    border: 2px dashed red !important;
```

Comment:
are we keeping the dashed border?

Reply:
I thought this was the plan?

Reply:
that was the plan when we were in dev mode. 
Do you think it looks good?